src/App.js

```diff
@@ -35,6 +36,8 @@ class App extends Component {
             component={ViewAnalytics}
           />{' '}
           {/* View Analytics Redirect */}
+          <Route exact path="/Draft" component={Draft} />{' '}
+          {/* View draft Redirect */}
```

Comment:
there isn't a _problem_ with adding these comments, but they're also not necessary. I would suggest we remove them, or at least put them above the routes to match comment conventions

Reply:
Resolved